TKeazirian.HTTPServer/Constants.cs

```diff
@@ -5,8 +5,9 @@ public static class Constants
         public const string NewLine = "\r\n";
         public const string BodySeparator = "\r\n\r\n";
         public const string Space = " ";
-        public const string Status200 = "HTTP/1.1 200 OK";
-        public const string Status404 = "HTTP/1.1 404 Not Found";
-        public static string Status301 = "HTTP/1.1 301 Moved Permanently";
+        public const string HttpVersion = "HTTP/1.1";
+        public const string Ok = "OK";
+        public const string NotFound = "Not Found";
+        public const string Moved = "Moved Permanently";
```

Comment:
You introduce an `enum` for [HttpStatusCode](https://github.com/t-keazirian/HTTPServer/blob/84a9780b8f83a3f90ee876033851562e50c912ec/TKeazirian.HTTPServer/Response/HttpStatusCode.cs#L1-L8); these three "constants" appear to be a mapping from `HttpStatusCode` to `HttpStatusMessage` ... does it make sense to extract an `HttpStatusMessage` as well?  or just keep them all in this `Constants` class?

Reply:
It actually looks like you have done over half the work in this [StatusMessage](https://github.com/t-keazirian/HTTPServer/blob/84a9780b8f83a3f90ee876033851562e50c912ec/TKeazirian.HTTPServer/Response/StatusMessage.cs#L1-L23) class.

Reply:
I kept them in `Constants` because I'm still referring to them in the `StatusMessage` class, and using the constants in tests...but are you saying to complete this I would take them out of `Constants` altogether and manage them in the `StatusMessage` class alone?

Reply:
I pulled them out of `Constants` and put them into the `StatusMessage` class as class variables.